app/views/questions/index.html.erb

```diff
@@ -17,28 +17,27 @@
   <% collapse_count = 1 %>
 
   <div class = "question_list">
-  <% @questions.each do |question| %>
-
-      
+  <% @questions.each do |question| %>    
       <% if question['user_id'] == current_user.id %>
-      <div class = "panel-group my-question">
+    <div class = "panel-group my-question">
       <% else %>
-      <div class = 'panel-group other-question hidden'>
+    <div class = 'panel-group other-question hidden'>
       <% end %>
-
-
       <div class="panel panel-default">
         <div class="panel-heading">
           <h4 class="panel-title">
             <div class="row valign">
               <div class="col-sm-11">
-                <b><%= link_to question["text"], "/questions/#{question["id"]}" %></b>
+                <!-- Toggles the modal. Data that is included: question answer, question text, and question ID -->
+                <span style="cursor:default"><a data-toggle="modal" data-target="#question_display_Modal" data-answer="<%=question["answers"][0]["text"]%>" data-qid="<%=question["id"]%>" data-question="<%= question["text"]%>"><%= question["text"]%></span>
```

Comment:
In the future it would be easier to read if this `a` element spanned multiple lines of code, like this:
```
<a data-toggle="modal" 
  data-target="#question_display_Modal" 
  data-answer="<%=question["answers"][0]["text"]%>" 
  data-qid="<%=question["id"]%>" 
  data-question="<%= question["text"]%>">
    <%= question["text"]%>
</a>
```